tools/snapshot/node_mksnapshot.cc

```diff
@@ -58,15 +58,23 @@ int main(int argc, char* argv[]) {
 
 int BuildSnapshot(int argc, char* argv[]) {
   if (argc < 2) {
-    std::cerr << "Usage: " << argv[0] << " <path/to/output.cc>\n";
+    std::cerr << "Usage: " << argv[0]
+              << " [--use-string-literals] <path/to/output.cc>\n";
     std::cerr << "       " << argv[0] << " --build-snapshot "
               << "<path/to/script.js> <path/to/output.cc>\n";
     return 1;
   }
 
+  std::vector<std::string> args{argv, argv + argc};
+  size_t size_before_remove = args.size();
+  args.erase(std::remove(args.begin(), args.end(), "--use-string-literals"),
```

Comment:
This seems like a costly way of validating if it exists on a std::vector. Can we achieve the same thing with iterating over it, instead of mutating?

Reply:
Yeah that makes more sense 👍